.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    container: madninja/builder-erlang:3
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build
+      run: rebar3 compile
+
+    - name: Run Dialyzer
+      run: rebar3 dialyzer
+
+    - name: Run tests
+      run: rebar3 as test do ct
+
+    - name: Generate coverage report
+      run: rebar3 covertool generate
+
+    - name: Upload coverage report
+      uses: codecov/codecov-action@v1.0.7
```

Comment:
Huh, are you sure this really required the codecov tool in the container? affect, the codecov action is fully self contained.

Reply:
Look at Erlang-erasure.. It does _not_ require codecov installed in the image. I'd like to delete the :3 image from the repo since it adds a bunch of unneeded software

Reply:
It does not need the `codecov` binary in the image. When I first said it did, I didn't know this action was available and basing my request on the `make ci` target which does use the `codecov` binary.

Reply:
ah ok.. I'll delete tag 3 of that builder image then since it's not useful. Can you update this PR to use tag 2?

Reply:
> Look at Erlang-erasure.. It does not require codecov installed in the image.

True, it does not require `codecov`, and would not even it reported coverage, but it doesn't report coverage.